README.md

```diff
@@ -349,8 +349,8 @@ The custom configuration options for the Celery workers are listed below:
   `30s` (30 seconds).
 * `iib_total_attempts` - the total number of attempts to make at trying a function relating to the
   container registry before erroring out. This defaults to `5`. It's also used as the max number of attempts to buildah when receiving HTTP 50X errors.
-* `iib_retry_delay` - the delay in seconds between retry attempts. It's just used for buildah when receiving HTTP 50X errors. This defaults to `4`.
-* `iib_retry_jitter` - the extra seconds to be added on delay between retry attempts. It's just used for buildah when receiving HTTP 50X errors. This defaults to `2`.
+* `iib_retry_delay` - the delay in seconds between retry attempts. It's just used for buildah when receiving HTTP 50X errors. This defaults to `5`.
```

Comment:
Should we change this setting as well? It mentions buildah retries will be affected with this only

Reply:
The buildah retries happen too soon not allowing any network glitch to resolve. Hence I changed this too. Please let me know if I should revert it

Reply:
Nope. I don't mind this change, we can definitely keep it. :)
I wasn't aware that buildah retries also needed to be optimized.